docs/source/tutorials/ATL03/IceSat_ATL03_Access.ipynb

```diff
@@ -0,0 +1,781 @@
+{
```

Comment:
<p>Recommend adding the expansion for (ATL03) before the parentheses.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>